README.md

```diff
@@ -1,4 +1,16 @@
-## Scanning the transcriptome for ago2:miRNA to mRNA binding sites and predicting fold change
+# Scanning the transcriptome for ago2:miRNA to mRNA binding sites and predicting fold change
 
-### Data TargetScan 2015 7miRs HCT116 transfection
-https://drive.google.com/drive/folders/15pX8qrauPJbKYo6r07jRtBLZaRTU6LdO?usp=sharing
+## Running example models and evaluation
+### Set up the environment
+Create a conda env by running ```conda create --name YOUR_ENVIRONMENT_NAME --file spec-file.txt```
+
+Activate an existing environment by ```conda activate YOUR_ENVIRONMENT_NAME```
+
+### Download the data
+Run ```./data/download_data_for_ML.sh``` which installs the ```gdown``` package (if not installed yet) through pip and downloads the datasets.
```

Comment:
There is no `./data/download_data_for_ML.sh` script.

Reply:
Moved